src/test/regress/multi_schedule

```diff
@@ -17,6 +17,7 @@
 # ---
 test: multi_extension
 test: multi_table_ddl
+test: multi_schema_support
```

Comment:
We should also add multi_schema_support to multi_task_tracker_extra_schedule. Read the file comments above.